src/sidebar/templates/annotation.html

```diff
@@ -29,26 +29,23 @@
     text="vm.state().text">
   </annotation-body>
 
+  <div class="annotation-link-more">
```

Comment:
the `div` is new, but its wrapping existing code which was from line 46 in the original .html file. A needed to add the dive to get different styling for this link. Before it was taking up an entire column, but I want it to just float on the right hand side and also take up no space when it's not rendered. The class name is crappy. I didnt want to change the name of the child class `annotation-link` which is terribly vague -- I think its due for some css renaming when we get there, but I needed to add a wrapper around it. I thought that annotation-link-wrapper was also a bad name. I also considering calling it 

annotation-link__wrapper
annotation-link__a

or something to that effect, but its terrible and also required renaming the `<a>` class which I wanted to avoid this round. So I needed up with annotation-link-more which at least add some specificity to what it is.